presto-jdbc/src/main/java/com/facebook/presto/jdbc/ConnectionProperties.java

```diff
@@ -170,6 +172,15 @@ public DisableCompression()
         }
     }
 
+    private static class DisableHttp2
+            extends AbstractConnectionProperty<Boolean>
+    {
+        public DisableHttp2()
+        {
+            super("disableHttp2", NOT_REQUIRED, ALLOWED, BOOLEAN_CONVERTER);
```

Comment:
Thanks for making the changes! I like the approach and the ability to accept the concise protocol names. In preparation to get these changes merge ready, check out [the wiki section on commit formatting and pull requests](https://github.com/prestodb/presto/wiki/Review-and-Commit-guidelines#commit-formatting-and-pull-requests), in particular- we want the commits that we merge to be a consumable changelog, so that means rebasing commits instead of merging from master as well as squashing commits into their logical atomic changes. In this context, I think the commits currently present could be reasonably squashed into a single commit (be sure to check out the linked [commit message guidelines](https://chris.beams.io/posts/git-commit/) too!)

Regarding whether to add the CLI / Benchmark options in a separate PR, it depends. If you're willing to do the work to add those changes, it would be easier to review and merge them as a single PR with those additional changes in their own commits. If you don't have the time to make the extra changes required to do that, it's totally understandable and in that case we can talk about just merging the change to the JDBC driver for now and see if someone else can pick up the task of adding support to the CLI and Benchmark options later.

Reply:
No problem, thank you for reviewing them! I just rebased and force pushed squashing all the changes.

I don't know that I'll have time right now to address CLI + Benchmark changes but I can consider making them at a later date.